ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/api/data_api.py

```diff
@@ -33,6 +33,630 @@ class DataApi(ApiBase):
     Ref: https://github.com/swagger-api/swagger-codegen
     """
 
+    def export_binary_data_for_latest(
+        self,
+        *,
+        database_key: "str",
+        record_history_guid: "str",
+        attribute_guid: "str",
+        mode: "Optional[str]" = None,
+        x_ansys_vc_mode: "Optional[str]" = None,
+        max_width: "Optional[int]" = None,
+        max_height: "Optional[int]" = None,
+    ) -> "Union[None, str]":
+        """Get the binary data for a given attribute and the latest version of a record history in given database. Must be a file or picture attribute.
+
+        This method makes a synchronous HTTP request.
+
+        Parameters
+        ----------
+        database_key: str
+        record_history_guid: str
+        attribute_guid: str
+        mode: str
+            The version control mode. If not provided, defaults to write mode if the user is allowed to see that. Can also be set in the header.
+        x_ansys_vc_mode: str
+            The version control mode. If not provided, defaults to write mode if the user is allowed to see that. Can also be set in the query string.
```

Comment:
```suggestion
            Version control mode. If a mode is not provided, the default is the write mode if the user is allowed to see this mode. The mode can also be set in the header.
        x_ansys_vc_mode: str
            Version control mode. If a mode is not provided, the default is the write mode if the user is allowed to see this mode. The mode can also be set in the query string.
```
I'm not sure why these two parameters are almost identical. I assume that you will be adding descriptions and defaults later.